python/.changelog.d/+13d8ef2d.changed

```diff
@@ -0,0 +1 @@
+Internal names are used consistently in constants and names. Original model-based names are kept as aliases for backwards compatibility.
```

Comment:
Why does the file have such a strange name? (`+13d8ef2d.changed`)

I would expect `3349.changed`.

Does not apply when you plan to rename it after having PR number :)

Reply:
when you run `towncrier create +.changed`, it will generate a name like this. then there is no linked issue in the changelog. i can rename it, it doesn't seem to useful though

Reply:
Not insisting on the renaming, was just curious